### ISSUE TITLE
Unity 2018.2.6f1, Xcode 10.1  with GVR

### ISSUE THREAD

**Issue by msk-psp** [jiulongw/swift-unity#122]
<img width="1837" alt="2019-02-13 4 06 02" src="https://user-images.githubusercontent.com/23733661/52693225-762ab680-2fa9-11e9-9109-2322a51deef0.png">

I built iOS project using GVR and I want to embed it, so I followed steps in Unity.xcconfig Jiulongw left

I think I followed steps well, but It occur an error when I build it

would tell me what is wrong?

**Comment by msk-psp** [comment]
added manually GVRSDK including headers and add `-Wl,-undefined,dynamic_lookup`  at Other Linker Flags

and I finally encountered this error 

dyld: Symbol not found: _AppControllerClassName
  Referenced from: /Users/username/Library/Developer/CoreSimulator/Devices/6CB33773-A461-485B-8D45-B19A73746486/data/Containers/Bundle/Application/1DEF88AF-96CC-46A1-AF34-40AA1AF5F3F7/projectname.app/projectname
  Expected in: flat namespace
 in /Users/username/Library/Developer/CoreSimulator/Devices/6CB33773-A461-485B-8D45-B19A73746486/data/Containers/Bundle/Application/1DEF88AF-96CC-46A1-AF34-40AA1AF5F3F7/projectname.app/projectname

**Comment by msk-psp** [comment]
I just succeeded to run iOS project embeded Unity project using GVR

what I have dones:

1. manually add GVRSDK into iOS project (for an error 'library not found for -lGVRSDK)
2. add flags -Wl, -undefined,dynamic_lookup ( for an error "sysmbol(s) not found for architecture arm64" started with '"_SetUnityiOS81orNewerBody", referenced from:.....blahblah')
3. build for device from Unity build settings ( I failed running with simulator)
4. manually add GVRSDK headers (GVROverlayView.h not found)
5. followed steps in Unity.xcconfig which is introduce "If your unity project contains dependencies that uses cocoapod(e.g GVR SDK)" (i think this is not working properly so that Xcode occur the errors above) 
6. constantly clean build & folder clean when add some files
7. version up Unity Editor to 2018.3.5f1 ( idk this is essential)
